tsdb/querier.go

```diff
@@ -721,16 +729,17 @@ type blockSeriesSet struct {
 	blockBaseSeriesSet
 }
 
-func newBlockSeriesSet(i IndexReader, c ChunkReader, t tombstones.Reader, p index.Postings, mint, maxt int64) storage.SeriesSet {
+func newBlockSeriesSet(i IndexReader, c ChunkReader, t tombstones.Reader, p index.Postings, mint, maxt int64, trimEnabled bool) storage.SeriesSet {
```

Comment:
How about this (same for newBlockChunkSeriesSet)? Easier to read and understand the code. Else it's a mental effort to see the hint as disabled, and remap everything in mind to consider it enabled elsewhere in code.

```suggestion
func newBlockSeriesSet(i IndexReader, c ChunkReader, t tombstones.Reader, p index.Postings, mint, maxt int64, trimDisabled bool) storage.SeriesSet {
```